lib/utils.js

```diff
@@ -65,10 +65,13 @@ function parseOutput(lines, inputFilename, pathPrefix) {
             const lineObj = {text: line};
             const match = line.replace(/\x1b\[[\d;]*[mK]/g, '').match(re);
             if (match) {
+                var text = match[5].trim();
+                if (match[1])
+                    text = match[1].toLowerCase() + ": " + text;
```

Comment:
The ugly part of this is that it will probably upset output formatting under some circumstances where the compiler expected this line to be `match[1].toLowerCase().length` characters less.

Reply:
I realy don't understand what you mean by that...
However: I think making the above regex more specific will reduce the chances of any mishaps here as well

Reply:
I meant cases where te compiler does something like
```
<source:1>: d is not defined
return 1 + d;
         ~~^~~
```
I'm just worried that something might get in front of the `return 1 + d;` line, making the `~~^~~` icons be missplaced. I know it's probably not worth the trouble, just something that came to my mind